src/components/ImportForm/SourceSection/SourceSection.tsx

```diff
@@ -52,27 +48,27 @@ export const SourceSection: React.FC<SourceSectionProps> = ({
 
   const fieldId = getFieldId('source', 'input');
   const isValid = !(touched && error);
-  const label = gitOnly ? 'Git repo URL' : 'Git repo URL or container image';
+  const label = 'Git repo URL';
 
   const [{ isGit, isRepoAccessible, serviceProvider, accessibility }, accessCheckLoaded] =
     useAccessCheck(sourceUrl, authSecret);
 
   const setFormValidating = React.useCallback(() => {
     setValidated(ValidatedOptions.default);
-    setHelpText('Validating...');
+    setHelpText('Validating authentication...');
     setFieldValue('isValidated', false);
   }, [setFieldValue]);
 
   const setFormValidated = React.useCallback(() => {
     setValidated(ValidatedOptions.success);
-    setHelpText('Validated');
+    setHelpText('Validated authentication');
     setFieldValue('isValidated', true);
   }, [setFieldValue]);
 
   const handleSourceChange = React.useCallback(() => {
     const searchTerm = source;
     const isGitUrlValid = gitUrlRegex.test(searchTerm?.trim());
-    const isContainerImageValid = !gitOnly && containerImageRegex.test(searchTerm);
+    const isContainerImageValid = containerImageRegex.test(searchTerm);
```

Comment:
I can still enter URLs for `quay.io` and validation succeeeds.

Reply:
Updated.